wagtail/models/i18n.py

```diff
@@ -210,6 +220,26 @@ def check(cls, **kwargs):
                 )
             )
 
+        if is_translation_model and not any(
+            isinstance(constraint, models.UniqueConstraint)
+            and set(constraint.fields) == {"translation_key", "locale"}
+            for constraint in constraints
+        ):
+            errors.append(
+                checks.Warning(
+                    "{}.{} is missing a UniqueConstraint for the fields: {}. Consider using models.UniqueConstraint.".format(
+                        cls._meta.app_label, cls.__name__, unique_constraint_fields
+                    ),
```

Comment:
We can use [`._meta.label`](https://docs.djangoproject.com/en/4.2/ref/models/options/#django.db.models.Options.label), and use f-string. Also, the "Consider using" ... isn't needed, because we have the `hint` separately.

```suggestion
                    f"{cls._meta.label} is missing a UniqueConstraint for the fields: {unique_constraint_fields}.",
```